### ISSUE TITLE
last child of xcyl, ycyl, and zcyl ignored (first child doubled)

### ISSUE THREAD

**Issue by adrianVmariano** [revarbat/BOSL2#99]
It appears that the last child of xcyl, ycyl and zcyl is ignored.  Also I think the first child is run twice.   Example:

```
xcyl(r=2, h=8){
   attach(TOP) cube([1,1,1],anchor=BOTTOM);
   attach(BOTTOM)    cube([1,1,1],anchor=BOTTOM);
}
```

If I comment out either line then the other one performs as expected.  If I add a third child I get both cubes.

**Comment by revarbat** [comment]
Bug is that each of the [xyz]cyl() modules are calling `children(0)` instead of `children($children-1)` like they should.

**Comment by revarbat** [comment]
Fixed in 91967792576e58a2a4414f890558cd87a52cb913
